tracer/src/Datadog.Trace/LibDatadog/NativeInterop.cs

```diff
@@ -0,0 +1,67 @@
+// <copyright file="NativeInterop.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+#nullable enable
+
+using System;
+using System.Runtime.InteropServices;
+
+namespace Datadog.Trace.LibDatadog;
+
+internal class NativeInterop
+{
+    private const string DllName = "LibDatadog";
```

Comment:
/question

shouldn't this be name of file?

Reply:
in this case, it can be whatever you want: here I use libdatadog for documentation. 
Since it will be rewritten by the native side, it does not matter.